lib/gui/app/modules/analytics.ts

```diff
@@ -15,84 +15,54 @@
  */
 
 import * as _ from 'lodash';
-import * as resinCorvus from 'resin-corvus/browser';
-
-import * as packageJSON from '../../../../package.json';
-import { getConfig } from '../../../shared/utils';
+import { Client, createClient, createNoopClient } from 'analytics-client';
+import * as Sentry from '@sentry/electron/renderer';
 import * as settings from '../models/settings';
 import { store } from '../models/store';
+import * as packageJSON from '../../../../package.json';
+import fetch from 'electron-fetch';
 
-const DEFAULT_PROBABILITY = 0.1;
-
-async function installCorvus(): Promise<void> {
-	const sentryToken =
-		(await settings.get('analyticsSentryToken')) ||
-		_.get(packageJSON, ['analytics', 'sentry', 'token']);
-	const mixpanelToken =
-		(await settings.get('analyticsMixpanelToken')) ||
-		_.get(packageJSON, ['analytics', 'mixpanel', 'token']);
-	resinCorvus.install({
-		services: {
-			sentry: sentryToken,
-			mixpanel: mixpanelToken,
-		},
-		options: {
-			release: packageJSON.version,
-			shouldReport: () => {
-				return settings.getSync('errorReporting');
-			},
-			mixpanelDeferred: true,
-		},
-	});
-}
-
-let mixpanelSample = DEFAULT_PROBABILITY;
-
+let analyticsClient: Client;
 /**
  * @summary Init analytics configurations
  */
-async function initConfig() {
-	await installCorvus();
-	let validatedConfig = null;
-	try {
-		const configUrl = await settings.get('configUrl');
-		const config = await getConfig(configUrl);
-		const mixpanel = _.get(config, ['analytics', 'mixpanel'], {});
-		mixpanelSample = mixpanel.probability || DEFAULT_PROBABILITY;
-		if (isClientEligible(mixpanelSample)) {
-			validatedConfig = validateMixpanelConfig(mixpanel);
-		}
-	} catch (err) {
-		resinCorvus.logException(err);
+const initConfig = _.once(async () => {
+	const dsn =
+		(await settings.get('analyticsSentryToken')) ||
+		_.get(packageJSON, ['analytics', 'sentry', 'token']);
+
+	if (dsn) {
+		Sentry.init({ dsn });
 	}
-	resinCorvus.setConfigs({
-		mixpanel: validatedConfig,
-	});
-}
+
+	const projectName =
+		(await settings.get('analyticsAmplitudeToken')) ||
+		_.get(packageJSON, ['analytics', 'mixpanel', 'token']);
+
+	(global as any).fetch = fetch;
+	const clientConfig = {
+		projectName,
+		endpoint: 'data.balena-staging.com',
```

Comment:
@otaviojacobi is it a fixed value?

Reply:
I see now this is a draft PR for now.
Did not want to make anyone hurry, just got lost between the 50+ notifications to be cleared 😅 
We can hop on a call if you need help with electron any time.